web/client/plugins/TOC/components/__tests__/StyleBasedWMSJsonLegend-test.jsx

```diff
@@ -89,4 +92,74 @@ describe('test StyleBasedWMSJsonLegend module component', () => {
         expect(legendRuleElem).toBeTruthy();
         expect(legendRuleElem.length).toEqual(2);
     });
+    it('tests legend with empty rules', async() => {
```

Comment:
Are the tests working with async? Shouldn't be use the done callback?

Reply:
Yeah it works fine